salt/utils/platform.py

```diff
@@ -196,3 +196,11 @@ def is_photonos():
         x.strip('"').strip("'") for x in linux_distribution()
     ]
     return osname == "VMware Photon OS"
+
+
+@real_memoize
+def is_aarch64():
+    """
+    Simple function to return if host is AArch64 or not
+    """
+    return sys.platform.startswith("aarch64")
```

Comment:
this will end up being linux.  you need platform.machine or something.

Reply:
nice catch, fixed